cmd/zpool/zpool_main.c

```diff
@@ -6467,6 +6467,15 @@ status_callback(zpool_handle_t *zhp, void *data)
 		    "to be recovered.\n"));
 		break;
 
+	case ZPOOL_STATUS_IO_FAILURE_MMP:
+		(void) printf(gettext("status: The pool is suspended because "
+		    "multihost writes failed or were delayed, and another "
```

Comment:
To my ear the phrasing here doesn't flow quite right, maybe replace ", and another system..." with "; another system...".

Reply:
Done